client/src/pages/Payment/Payment.tsx

```diff
@@ -0,0 +1,192 @@
+
+  useEffect(() => {
+    getUserPaymentProfiles();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [loggedInUser]);
+
+  const getUserPaymentProfiles = async (): Promise<void> => {
+    updateLoggedInUser(loggedInUser);
+    const stripeId = await handleGetUserIds();
+    if (stripeId) {
+      handleGetPaymentProfiles(stripeId);
+    }
+  };
+
+  const handleGetUserIds = async (): Promise<any> => {
+    const user: any = loggedInUser;
+    try {
+      const { stripeId } = await getUserStripeId(user.id);
+      setUserIds({ stripeId, userId: user.id });
+      return stripeId;
+    } catch (err) {
+      const error: any = err;
+      alert(error.message);
+    }
+  };
+
+  const handleGetPaymentProfiles = async (stripeId: any): Promise<void> => {
+    try {
+      const response = await retrievePaymentProfiles(stripeId);
+      if (response) {
+        updatePaymentProfiles(response.paymentMethods.data);
+        setDefaultPaymentMethodId(response.customer.invoice_settings.default_payment_method);
+      }
+
+      SetPaymentProfileExist(true);
+    } catch (err) {
+      const error: any = err;
+      alert(error.message);
+    }
+  };
+
+  const handleDefaultPaymentMethod = async (PaymentMethodId: string): Promise<void> => {
+    const response = await setDefaultPaymentProfile(PaymentMethodId, userIds.stripeId);
+    setDefaultPaymentMethodId(response.invoice_settings.default_payment_method);
+  };
+
+  const handleCardExpMonth = (month: string): string => {
+    return month.length > 1 ? month : '0' + month;
+  };
+  const handleCardExpYear = (year: string): string => {
+    return year.toString().slice(2);
+  };
+
+  const handleOpenDialog = () => {
+    setOpen(true);
+  };
+  const label = { inputProps: { 'aria-label': 'Checkbox demo' } };
+
+  const handleCloseDialog = () => {
+    setOpen(false);
+  };
+
+  return (
+    <Grid container className={classes.container}>
+      <Grid item xs={12} sm={9} md={8} lg={7} className={classes.cardContainer}>
+        <Card className={classes.card}>
+          <Grid item>
+            <Typography variant="h5" className={classes.h6} align="center">
+              {' '}
+              Payment Methods
+            </Typography>
+          </Grid>
+          <Grid item container md={10} className={classes.gridContainer}>
+            <Typography variant="h6" className={classes.containerInfo}>
+              {paymentProfiles && 'Saved Payment Profiles:'}
+              {!paymentProfiles && "You don't have any Payment Profile"}
+            </Typography>
+            <Grid item container spacing={2} className={classes.grid}>
+              {paymentProfiles &&
+                paymentProfiles.map((paymentMethod: any) => (
+                  <Grid item xs={12} sm={12} md={6} key={paymentMethod.id}>
+                    <Card className={classes.payCard}>
+                      <div className={classes.cardHeader}>
```

Comment:
Please use MUI `Box` components instead of HTML divs. A lot of the CSS properties can be passed as props to the Box component

Eg: `<Box display="flex" alignItems="center".....>`